presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -763,6 +764,35 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         return TableStatistics.empty();
     }
 
+    @Override
+    public void createSchema(JdbcIdentity identity, String schemaName)
+    {
+        String sql = "CREATE SCHEMA " + quoted(schemaName);
+
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            execute(connection, sql);
+        }
+        catch (SQLException e) {
+            throw new PrestoException(JDBC_ERROR, e);
+        }
+    }
+
+    @Override
+    public void dropSchema(JdbcIdentity identity, String schemaName)
```

Comment:
We need a feature toggle for this.
Even if long term plan is to support file-based authZ, we need backward compatibility.

I am not sure it's safe to enable this by default.
A schema might be associated with some configuration and drop/recreate may effectively change this configuration. (Like "location" in Hive, or maybe tablespace in a traditional database).
I'd rather have it off by default.

cc @ebyhr @electrum